app/flatpak-builtins-build-bundle.c

```diff
@@ -177,6 +173,12 @@ iterate_bundle_icons (GFile                     *root,
   g_autoptr(GFile) icons_dir =
     g_file_resolve_relative_path (root,
                                   "files/share/app-info/icons/flatpak");
+  if ( g_file_query_file_type (icons_dir, 0, NULL) != G_FILE_TYPE_DIRECTORY) 
+  {
```

Comment:
Indentation is wrong, also, move this code after declaring the rest of the variables.

Reply:
would you mind checking now

Reply:
Need two spaces before the bracket, and the code has not moved.